### ISSUE TITLE
Unable to create rules with `web` submodule

### ISSUE THREAD

**Issue by Vrtak-CZ** [terraform-aws-modules/terraform-aws-security-group#286]
## Description

Failed to create rules in SG with Web submodule.

- [x] ✋ I have searched the open/closed issues and my issue is not listed.

## Versions

- Module version [Required]: `4.17.2`

- Terraform version: `1.4.2`
- Provider version(s): _AWS_ `4.67.0`

## Reproduction Code [Required]

```terraform
module "lb_sg" {
  source  = "terraform-aws-modules/security-group/aws//modules/web"
  version = "4.17.2"

  name   = "LoadBalancer"
  vpc_id = module.vpc_utils.vpc_id
}
```

Just run the example code

## Expected behavior

Creates SG with ingress for 80, 8080, 443.

## Actual behavior

```
╷
│ Error: waiting for Security Group (sg-021eb2d9c6554f57e) Rule (sgrule-3790310160) create: couldn't find resource
│ 
│   with module.lb_sg.module.sg.aws_security_group_rule.ingress_rules[2],
│   on .terraform/modules/lb_sg/main.tf line 66, in resource "aws_security_group_rule" "ingress_rules":
│   66: resource "aws_security_group_rule" "ingress_rules" {
│ 
╵
╷
│ Error: waiting for Security Group (sg-021eb2d9c6554f57e) Rule (sgrule-3200772996) create: couldn't find resource
│ 
│   with module.lb_sg.module.sg.aws_security_group_rule.ingress_rules[1],
│   on .terraform/modules/lb_sg/main.tf line 66, in resource "aws_security_group_rule" "ingress_rules":
│   66: resource "aws_security_group_rule" "ingress_rules" {
│ 
╵
╷
│ Error: waiting for Security Group (sg-021eb2d9c6554f57e) Rule (sgrule-975813690) create: couldn't find resource
│ 
│   with module.lb_sg.module.sg.aws_security_group_rule.ingress_rules[0],
│   on .terraform/modules/lb_sg/main.tf line 66, in resource "aws_security_group_rule" "ingress_rules":
│   66: resource "aws_security_group_rule" "ingress_rules" {
│ 
╵
╷
│ Error: waiting for Security Group (sg-021eb2d9c6[554](https://gitlab.com/mailstep/infra/20-new/utils-terraform-global/-/jobs/4444528595#L554)f57e) Rule (sgrule-3717443498) create: couldn't find resource
│ 
│   with module.lb_sg.module.sg.aws_security_group_rule.ingress_rules[3],
│   on .terraform/modules/lb_sg/main.tf line 66, in resource "aws_security_group_rule" "ingress_rules":
│   66: resource "aws_security_group_rule" "ingress_rules" {
│ 
╵
```

### Terminal Output Screenshot(s)

See above

## Additional context

no

**Comment by dsikes** [comment]
I also have the same issue, but mine is with the `https-443` submodule.

```
module "https_443_security_group" {
  source  = "terraform-aws-modules/security-group/aws//modules/https-443"
  version = "~> 5.0"
  name    = "${module.namespace.prefix}-https-443-sg"
  vpc_id  = module.vpc.vpc_id
}
```

ERROR:
```
 Error: waiting for Security Group (sg-*********) Rule (sgrule-********) create: couldn't find resource
│ 
│   with module.https_443_security_group.module.sg.aws_security_group_rule.ingress_rules[0],
│   on .terraform/modules/https_443_security_group/main.tf line 66, in resource "aws_security_group_rule" "ingress_rules":
│   66: resource "aws_security_group_rule" "ingress_rules" {
```

**Comment by Vrtak-CZ** [comment]
I found a problem and fixed it by adding `ingress_cidr_blocks = ["0.0.0.0/0"]`.

So result is 

```terraform
module "lb_sg" {
  source  = "terraform-aws-modules/security-group/aws//modules/web"
  version = "4.17.2"

  name   = "LoadBalancer"
  vpc_id = module.vpc_utils.vpc_id

  ingress_cidr_blocks = ["0.0.0.0/0"]
}
```
